tests/conftest.py

```diff
@@ -29,6 +31,18 @@ def generate_notification_status(published):
     return NotificationStatus.COMPLETED if published else NotificationStatus.WAITING
 
 
+@pytest.fixture(scope="session", autouse=True)
+def set_dynaconf_environment(request) -> None:
+    os.environ["ENV_FOR_DYNACONF"] = "testing"
+    os.environ["FORCE_ENV_FOR_DYNACONF"] = "testing"
+
+    def unset_env() -> None:
+        os.environ["ENV_FOR_DYNACONF"] = ""
+        os.environ["FORCE_ENV_FOR_DYNACONF"] = ""
+
+    request.addfinalizer(unset_env)
```

Comment:
I never used this.

It's more pythonic to use yield

Reply:
Me neither, I copied It from tortoise's setup. I'll look into yield